asv_bench/benchmarks/categoricals.py

```diff
@@ -1,114 +1,140 @@
-from .pandas_vb_common import *
-try:
-    from pandas.api.types import union_categoricals
-except ImportError:
-    try:
-        from pandas.types.concat import union_categoricals
-    except ImportError:
-        pass
```

Comment:
can you leave this try .. except in place? It is meant to be able to run the benchmarks with different older versions